### PR TITLE
[GG-2576] Build Map Component

### PR DESCRIPTION
**Pull request by sshalu12**
Build Map component

### CODE REVIEW

File: app/package.json [hunk 1]
```diff
@@ -1,13 +1,15 @@
 {
-  "name": "app",
+  "name": "reactt",
```
Comment (ashutoshgt) [inline app/package.json]: why did we change the name?

File: app/public/index.html [hunk 2]
```diff
@@ -2,33 +2,23 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```
Comment (ashutoshgt) [inline app/public/index.html:5]: why did we remove this?

File: app/public/index.html [hunk 3]
```diff
@@ -2,33 +2,23 @@
...
-    -->
-    <title>React App</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.4.1/semantic.min.css"
+      integrity="sha512-8bHTC73gkZ7rZ7vpqUQThUDhqcNFyYi2xgDgPDHc+GXVGHXq+xPjynxIopALmOPqzo9JZj0k6OqqewdGO3EsrQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>Mapbox Data</title>
```
Comment (ashutoshgt) [inline app/public/index.html]:
```suggestion
    <title>GeoVisualiser</title>
```

File: app/src/index.css [hunk 4]
```diff
@@ -0,0 +1,4 @@
+.map-container {
+    height: 1100px;
+  }
+
```
Comment (ashutoshgt) [inline app/src/index.css]: remove extra lines and add a single new line at end of file (eof)

**Review by ashutoshgt — CHANGES_REQUESTED** [review]

File: app/src/state.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
+import React, { useRef, useEffect } from "react";
+import mapboxgl from "!mapbox-gl"; // eslint-disable-line import/no-webpack-loader-syntax
+import Header from "./Header";
+
+export default function State() {
+  const mapContainer = useRef(null);
+  const map = useRef(null);
+  useEffect(() => {
+    if (map.current) return; // initialize map only once
```
Comment (gtvikas) [inline app/src/state.js]: @ashutoshgt as i understand whenever we are clicking state to poi or poi to state, components are getting rendered completely, which will have `map.current` set to null. What is the need of this condition in current scenario

**Review by ashutoshgt — APPROVED** [review]

**Pull request merged**